### PR TITLE
Fix go.mod for v9

### PR DESCRIPTION
**Pull request by jsafrane**
It must announce v9 and not v8

Fixes: #137

### CODE REVIEW

File: examples/hostpath-provisioner/hostpath-provisioner.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 	"path"
 	"syscall"
 
-	"sigs.k8s.io/sig-storage-lib-external-provisioner/v8/controller"
+	"sigs.k8s.io/sig-storage-lib-external-provisioner/v9/controller"
```
Comment (bertinatto) [inline examples/hostpath-provisioner/hostpath-provisioner.go:27]: There are more references to `v8` in the `go.mod` file in this directory.

Comment (jsafrane) [reply examples/hostpath-provisioner/hostpath-provisioner.go:27]: Fixed

**Pull request merged**